securitycenter/clientv2/create_client_with_endpoint.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clientv2
+
+// [START securitycenter_set_client_endpoint_v2]
+import (
+	"context"
+	"fmt"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv2"
+	"google.golang.org/api/option"
+)
+
+// createClientWithEndpoint creates a Security Command Center client for a
+// regional endpoint, along with another client for the default endpoint.
+func createClientWithEndpoint(repLocation string) error {
```

Comment:
issue: pass in `io.Writer` object.

See [CONTRIBUTING](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#print-to-an-iowriter-for-snippets).

Reply:
Please see my previous comment:

> For both clients, it would be nice if this code could print the endpoint that each client is using to an io.Writer, so that the test could verify that the endpoints are correct. However, it looks like the Go client doesn't expose the endpoint that it's currently using.
>
> If there's a way to get a client's endpoint, please let me know!

Reply:
@telpirion Please advise.

Reply:
I don't believe there is an easy way to do that. I'll experiment on my own to check.